osidb/sync_manager.py

```diff
@@ -85,9 +99,25 @@ def schedule(cls, sync_id, *args, schedule_options=None, **kwargs):
         if schedule_options is None:
             schedule_options = {}
 
-        cls.objects.get_or_create(sync_id=sync_id)
+        created = cls.objects.get_or_create(sync_id=sync_id)[1]
         cls.objects.filter(sync_id=sync_id).update(last_scheduled_dt=timezone.now())
 
+        if not created and cls.MODE == cls.SyncManagerMode.EXCLUSIVE:
+            # If the task is already scheduled, skip
+            if not cls.is_scheduled(sync_id):
+                logger.info(
+                    f"{cls.__name__} {sync_id}: Task already in progress"
+                    f", postponing for {cls.COUNTDOWN} seconds"
+                )
+                if "countdown" not in schedule_options:
+                    schedule_options["countdown"] = cls.COUNTDOWN
```

Comment:
I think this can still produce incorrect message, as in the log message the `cls.COUNTDOWN` is used, but `countdown` can already contained in the `schedule_options` so there should be something like
```python
logger.info(
    f"{cls.__name__} {sync_id}: Task already in progress"
    f", postponing for {schedule_options.get("countdown") or cls.COUNTDOWN} seconds"
)

Reply:
Oh you're right, 🤔 I remember changing it after your comment, maybe the it got lost after rebasing when I updated rhubarb